src/MoBi.Core/Domain/Model/MoBiContext.cs

```diff
@@ -121,6 +121,19 @@ public T Create<T>(string id) where T : class, IObjectBase
          return id != null ? ObjectBaseFactory.Create<T>(id) : ObjectBaseFactory.Create<T>();
       }
 
+      public ICommand<MoBiContext> RunCommand(ICommand<MoBiContext> command)
+      {
+         if (command.IsAnImplementationOf<BuildingBlockChangeCommandBase>() && !shouldRunModuleConversionCommand(command))
```

Comment:
Actually, in this checking stage, we'll have to recurse all commands within a macro command, but you get the point